src/bakery_manager_mobile/lib/pages/add_task_page.dart

```diff
@@ -135,23 +154,39 @@ class AddTaskPageState extends State<AddTaskPage> {
       ),
       body: FutureBuilder<List<Recipe>>(
         future: _futureRecipes,
-        builder: (context, snapshot) {
-          if (snapshot.connectionState == ConnectionState.waiting) {
+        builder: (context, recipeSnapshot) {
+          if (recipeSnapshot.connectionState == ConnectionState.waiting) {
             return const Center(child: CircularProgressIndicator());
-          } else if (snapshot.hasError) {
-            return Center(child: Text('Error: ${snapshot.error}'));
-          } else if (!snapshot.hasData || snapshot.data!.isEmpty) {
+          } else if (recipeSnapshot.hasError) {
+            return Center(child: Text('Error: ${recipeSnapshot.error}'));
+          } else if (!recipeSnapshot.hasData || recipeSnapshot.data!.isEmpty) {
             return const Center(child: Text('No recipes found'));
           } else {
-            return _buildForm(context, snapshot.data!);
+            return FutureBuilder<List<Account>>(
+              future: _futureUsers,
+              builder: (context, userSnapshot) {
+                if (userSnapshot.connectionState == ConnectionState.waiting) {
+                  return const Center(child: CircularProgressIndicator());
```

Comment:
Could we remove this waiting bar? (or it can be put in a future commit)

Reply:
Removed